mod/follow.php

```diff
@@ -163,29 +163,22 @@ function follow_content(App $a)
 
 	$o = Renderer::replaceMacros($tpl, [
 		'$header'        => DI::l10n()->t('Connect/Follow'),
-		'$desc'          => '',
 		'$pls_answer'    => DI::l10n()->t('Please answer the following:'),
-		'$does_know_you' => ['knowyou', DI::l10n()->t('Does %s know you?', $ret['name'])],
-		'$add_note'      => DI::l10n()->t('Add a personal note:'),
-		'$page_desc'     => '',
-		'$friendica'     => '',
-		'$statusnet'     => '',
-		'$diaspora'      => '',
-		'$diasnote'      => '',
 		'$your_address'  => DI::l10n()->t('Your Identity Address:'),
-		'$invite_desc'   => '',
-		'$emailnet'      => '',
+		'$url_label'     => DI::l10n()->t('Profile URL'),
+		'$keywords_label'=> DI::l10n()->t('Tags:'),
```

Comment:
What are the tags for?

Reply:
They show the public tags of the profile you're about to follow.